src/app/features/exchange-set/ess-download-exchangeset/ess-download-exchangeset.component.scss

```diff
@@ -28,9 +32,21 @@
     margin: 0px 39px;
 }
 
-
 .f22{
```

Comment:
Please check the class name and font-size.  And for .f36 class also.

Reply:
resolved